presidio-anonymizer/api.py

```diff
@@ -3,11 +3,12 @@
 app = Flask(__name__)
```

Comment:
I think that flask is searching for app.py be default. And again, I vote for splitting app.py with routes.py api.py since both are about to expend